packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.stories.tsx

```diff
@@ -1,26 +1,36 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import { MockApp } from '../../../.storybook/components/MockApp';
-import PaymentLegalBlub from './index';
+import PaymentLegalBlurb from './index';
 import * as PaymentProvider from '../../lib/PaymentProvider';
+import { Meta } from '@storybook/react';
 
-storiesOf('components/PaymentLegalBlurb', module)
-  .add('default', () => (
-    <MockApp>
-      <PaymentLegalBlub {...{ provider: undefined }}></PaymentLegalBlub>
-    </MockApp>
-  ))
-  .add('paypal', () => (
-    <MockApp>
-      <PaymentLegalBlub
-        {...{ provider: PaymentProvider.PaymentProviders.paypal }}
-      ></PaymentLegalBlub>
-    </MockApp>
-  ))
-  .add('stripe', () => (
+export default {
+  title: 'components/PaymentLegalBlurb',
+  component: PaymentLegalBlurb,
+} as Meta;
+
+const storyWithContext = (
+  provider?: PaymentProvider.PaymentProvider,
+  storyName?: string
+) => {
+  const story = () => (
     <MockApp>
-      <PaymentLegalBlub
-        {...{ provider: PaymentProvider.PaymentProviders.stripe }}
-      ></PaymentLegalBlub>
+      <PaymentLegalBlurb {...{ provider }} />
     </MockApp>
-  ));
+  );
+
+  if (storyName) story.storyName = storyName;
+  return story;
+};
+
+export const Default = storyWithContext(undefined, 'default');
```

Comment:
Should this be `PaymentProvider.PaymentProviders.none` instead of `undefined`?

Reply:
You are correct. Thank you! Fixed!